src/index.js

```diff
@@ -200,6 +204,81 @@ class IpfsRepo {
   exists (callback) {
     this.version.exists(callback)
   }
+
+  /**
+   * Get repo status.
+   *
+   * @param {Object}  options
+   * @param {Boolean} options.human
+   * @param {function(Error, Object)} callback
+   * @return {void}
+   */
+  stat (options, callback) {
+    if (typeof options === 'function') {
+      callback = options
+      options = {}
+    }
+
+    options = Object.assign({}, {human: false}, options)
+
+    parallel([
+      (cb) => this.config.get('Datastore.StorageMax', (err, max) => {
+        if (err) {
+          cb(null, new Big(noLimit))
+        } else {
+          cb(null, new Big(max))
+        }
+      }),
+      (cb) => this.version.get(cb),
+      (cb) => this.blocks.query({}, (err, list) => {
+        list = list || []
+
+        const count = new Big(list.length)
+        let size = new Big(0)
+
+        list.forEach(block => {
+          size = size
+            .plus(block.value.byteLength)
+            .plus(block.key._buf.byteLength)
+        })
+
+        cb(err, {
+          count: count,
+          size: size
+        })
+      }),
+      (cb) => getSize(this.datastore, cb),
+      (cb) => getSize(this.keys, cb)
+    ], (err, results) => {
```

Comment:
Perhaps it would be easier to use the object notation of async/parallel, something like this:

```js
parallel({
  a: (cb) => cb(null, 1),
  b: (cb) => cb(null, 2),
  c: (cb) => cb(null, 3)
}, (err, results) => {
  if  (err) { handleError(err) }
  else {
    console.log(results) // { a: 1, b: 2, c: 3}
  }
})
```

Reply:
Yeah, it would probably get a bit more clear. I'll do that.